CODEOWNERS

```diff
@@ -25,23 +25,17 @@ CODEOWNERS @MrAlias @MadVikingGod @pellared
 detectors/aws/                                                          @open-telemetry/go-approvers @Aneurysm9
 detectors/gcp/                                                          @open-telemetry/go-approvers @dashpole
 
-exporters/autoexport                                                    @open-telemetry/go-approvers @MikeGoldsmith
+exporters/autoexport                                                    @open-telemetry/go-approvers @pellared
 
-instrumentation/github.com/astaxie/beego/otelbeego/                     @open-telemetry/go-approvers
```

Comment:
This one is already deprecated per https://github.com/open-telemetry/opentelemetry-go-contrib/pull/4092